src/mixins/legal-api-mixin.ts

```diff
@@ -30,16 +30,27 @@ export default class LegalApiMixin extends Vue {
    */
   async saveFiling (filing: IncorporationFilingIF, isDraft: boolean): Promise<any> {
     let filingId = this.getFilingId
+    let filingComplete
     // If have a filing id, update an existing filing
     if (filingId && filingId > 0) {
-      await this.updateFiling(filing, isDraft)
+      filingComplete = await this.updateFiling(filing, isDraft)
     } else {
       // Set the filingId to store
-      await this.createFiling(filing, isDraft)
+      filingComplete = await this.createFiling(filing, isDraft)
     }
 
-    // Complete a filing if not draft
-    if (!isDraft) await this.completeFiling(filing)
+    if (!isDraft && filingComplete) {
+      const paymentToken = filingComplete.header.paymentToken
+      const baseUrl = sessionStorage.getItem('BASE_URL')
+      const returnURL = encodeURIComponent(baseUrl + 'review-confirm?nrNumber=' +
```

Comment:
Return URL should be "draft incorporation UI", which is the Business Filings UI, which we haven't updated yet...
For now, I suggest returning to the businesses main page. (Anyway, returning to this UI is incorrect.)
... Unless you're thinking we come back here for now (while developing), as long as it doesn't break anything to file...?

Reply:
Bonehead move, we already discussed too. I will update to return to the businesses main page 👍 I don't think we anticipate coming back to the incorporations flow. If payment fails for example i think we still resume from the manage businesses page?

Reply:
Payment retries and resuming will be from the "draft incorporations UI"... next sprint. For now, the businesses main page is good.

Reply:
Ah right! We  can update once we have that piece :) Thanks!